packages/cli-repl/src/constants.ts

```diff
@@ -1,6 +1,12 @@
 import i18n from '@mongosh/i18n';
 import clr from './clr';
 
+export const TELEMETRY = `
+${i18n.__('cli-repl.cli-repl.telemetry')}
+${i18n.__('cli-repl.cli-repl.enableTelemetry')}${clr('enableTelemtry()', 'bold')}.
```

Comment:
I think your worst nightmare is here again :D

Reply:
it took me literally 20 min to understand what you meant :D